com.unity.ml-agents.extensions/package.json

```diff
@@ -5,7 +5,7 @@
   "unity": "2021.3",
   "description": "A source-only package for new features based on ML-Agents",
   "dependencies": {
-    "com.unity.ml-agents": "2.3.0-exp.3",
+    "com.unity.ml-agents": "2.3.0-exp.4",
```

Comment:
Just making sure... we advance this on main one version bump ahead of the release, no?